src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -321,6 +338,35 @@ const AccessibilityRequestDetailPage = () => {
           }}
         </Formik>
       </div>
+      <div
+        role="region"
+        aria-label="existing notes"
+        className="margin-top-6 margin-x-1"
+      >
+        <div
+          className="usa-sr-only"
+          id="AccessibilityRequestNotesTab-NotesList"
+        >
+          existing notes
+        </div>
+        <NotesList>
+          {data?.accessibilityRequest?.notes?.map(
+            note =>
+              note && (
+                <div key={note.id}>
```

Comment:
We can't have `div` here semantically since `NotesList` is  a list. Is there a reason why there's a `div`?

Reply:
I needed to add a key.  Is it ok to just add a key directly to the component `ListItem`?

Reply:
Yeah, it should be on `ListItem`.